ramls/item.json

```diff
@@ -395,6 +395,28 @@
       "id": "tags",
       "type": "object",
       "$ref": "raml-util/schemas/tags.schema"
+    },
+    "lastCheckIn": {
```

Comment:
Are these properties writable using this API?

Reply:
I guess not, thank you for catching.